### ISSUE TITLE
No consistent source of hassium during chapter that needs it

### ISSUE THREAD

**Issue by vladspellbinder** [al132mc/antimatter-chemistry#279]
So, in Chapter 4 you need to make the various data modules which need to blank ones which need asinine which needs hassium which...you can only get via random drops from loot crates or via fusion. With fusion being near the end of Chapter 5.

As of right now I've gotten a grand total of _seven_ hassium from the loot crates I've gotten, so I can't even make one blank plate.
I guess maybe I _have_ to set up some sort of mob farm using that drop of evil from Chapter 3? Because I'm not the sort to normally set up mob farms so I haven't done so yet.

I don't mind grinding for things, but I prefer to have at least one guaranteed source of something, even if I have to take a lot of steps to get there at least it's there. I'm still; a bit sore there isn't an easy source of slag here without having to be silly with anvils so hitting this bump on the hassium almost made me stop playing all together... .

**Comment by iradicated7** [comment]
I also have this problem with the pack, i just either grinded mobs for a while, just used the /give command...
would really like a change to make obtaining hassium much less impossible.

**Comment by SilverhuntX** [comment]
@vladspellbinder @iradicated7 Ideally I would recommend fully automated mob farms, as they will allow you to do something else, while loot crates are collected. At about the time the mob farm is introduced to chapter 4, is usually enough to acquire a decent amount of lootcrate ---> lots of hassium

**Comment by vladspellbinder** [comment]
In the end I did wind up making a mob farm, looking up some play throughs on how other people did it. I got plenty of hassium from the loot crates but didn't find it overly enjoyable, as I just personally don't like mob farms.
I dropped the mod shortly after getting the fission and fusion multiblocks and having gotten most of the data modules to perfect but just... didn't care any more really.
